interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/xscobot_moveit.launch.py

```diff
@@ -345,6 +351,17 @@ def generate_launch_description():
             description="the file path to the 'mode config' YAML file.",
         )
     )
+    declared_arguments.append(
```

Comment:
Why do we have two mode_configs?

Reply:
My bad that was a test edit where I tried passing 2 different mode config to see if it resolves the calibration issue with xscobot_moveit.